dtbase/webapp/app/users/templates/users.html

```diff
@@ -48,6 +58,17 @@ <h3>Create a user</h3>
       </form>
     </div>
   </div>
+  <script>
```

Comment:
If this is exactly the same as on the login page, could it be moved to `dtbase/webapp/app/base/static/javascript/utility.js` to avoid code duplication?

Reply:
It makes sense. Added to utility